portfolio.html

```diff
@@ -58,5 +58,176 @@ <h1 class="headline-heading">Hey there. I’m Norbi<br><span>I’m a software de
 
     </section>
 
+    <section class="work">
+        <h2 class="recent-works">My Recent Works</h2>
+        <img src="./images/mobile_version/Indicator/Indicator.png" class="indicator_horizontal" alt="indicator icon">
+
+        <div class="main-content-cards">
```

Comment:
- [ ] you still didn't fix the overflow in your website , kindly fix it as the previous reviewers requested 

![REVOVR](https://user-images.githubusercontent.com/89282221/185765664-afb32672-c311-4417-afd1-a6b28d845416.gif)